### ISSUE TITLE
iPhone horizontal view image crop

### ISSUE THREAD

**Issue by natdudley** [WikiNewZealand/pacific#57]
on iPhone horizontally, the image of the samoan lady crops poorly.

![IMG_0528](https://user-images.githubusercontent.com/3861245/129837072-48ef123c-824f-48a7-9cfa-4f268c51f2c4.PNG)

 It's unfortunate but not a blocking issue as rare to view site this way.
 
 Issue stems from max height of issue being set to % of viewport. I'll have a look at options.

**Comment by robisaacnz** [comment]
I have some (limited) scope to crop the source image we use differently, to help push it in the right direction, but we're using most of the image already.

**Comment by natdudley** [comment]
Yeah. because we had longer title I chose a portrait image which is what has caused this issue.

I could also play around with the flex proportions?

Maybe for viewport widths less than 850 (i.e., when vertical flex kicks in), I remove the viewport height restriction.

**Comment by natdudley** [comment]
Downside is it means you have a big image to scroll past, but maybe that's OK?

**Comment by natdudley** [comment]
Note: changes to this would also impact iPad pro in some aspect ratios. Need to test any changes there too.
